test/tests/meshgenerators/file_mesh_generator/tests

```diff
@@ -21,4 +21,25 @@
     design = 'meshgenerators/FileMeshGenerator.md'
     issues = '#14916'
   [../]
+
+  [./pre_checkpoint_load_test]
+    type = 'RunApp'
+    input = '2d_diffusion_test.i'
+    expect_out = 'Solve Converged'
+    cli_args = 'Outputs/checkpoint=true'
+    design = 'meshgenerators/FileMeshGenerator.md'
+    recover = false
+    issues = '#16192'
+  []
+
+  [./checkpoint_load_test]
+    type = 'Exodiff'
+    input = '2d_diffusion_test.i'
+    exodiff = '2d_diffusion_test_out.e'
+    cli_args = 'Mesh/inactive=square Mesh/fmg/type=FileMeshGenerator Mesh/fmg/file=2d_diffusion_test_out_cp/0001_mesh.cpr'
+    design = 'meshgenerators/FileMeshGenerator.md'
+    recover = false
+    prereq = pre_checkpoint_load_test
+    issues = '#16192'
```

Comment:
shouldn't we add min_parallel and max_parallel = something > 1 to make sure both tests are executed with the same number of cpu?

Reply:
no, because the parallel test target will use the same number of processors for all the test unless you specify to not to.